SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -126,14 +126,32 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 		}
 	});
 
-	LocalDeploymentManager->Init(GetOptionalExposedRuntimeIP());
```

Comment:
do we not have a runtime ip anymore?

Reply:
The IP is no longer needed at Init as we don't have to start the spatial service with it. Instead it's passed to `TryStartLocalDeployment` :D

Reply:
This was some missed cleanup